src/UnitTests/GitHub.App/Services/GitClientTests.cs

```diff
@@ -194,7 +194,11 @@ public class TheGetPullRequestMergeBaseMethod : TestBaseClass
             repo.Network.Remotes.Add(null, null).ReturnsForAnyArgs(remote);
             var gitClient = new GitClient(Substitute.For<IGitHubCredentialProvider>());
 
-            await gitClient.GetPullRequestMergeBase(repo, baseUri, headUri, baseSha, headSha, baseRef, headRef);
+            try
+            {
+                await gitClient.GetPullRequestMergeBase(repo, baseUri, headUri, baseSha, headSha, baseRef, headRef);
+            }
+            catch (NotFoundException) { }
```

Comment:
Ignoring an exception in a test is weird. Is it supposed to throw or not?

Reply:
It's certainly a bit unusual. The method it's calling would previously return a null when a comment couldn't be fetched/found. It now throws an exception.

The test is to check whether or not `Fetch` was called. It doesn't care about the exception or exception message. There is another test for that.

Reply:
Ah ok, in that case we should probably document that with a comment, so we know in a year from now what the assumption here is.